app/builders/switch_to_daily_digest_email_builder.rb

```diff
@@ -0,0 +1,46 @@
+class SwitchToDailyDigestEmailBuilder
+  def initialize(subscriber, subscriptions)
+    @subscriber = subscriber
+    @subscriptions = subscriptions
+  end
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def call
+    Email.create!(
+      address: subscriber.address,
+      subject: "Your GOV.UK email subscriptions",
+      body: body,
+      subscriber_id: subscriber.id,
+      created_at: Time.zone.now,
+      updated_at: Time.zone.now,
+    ).id
+  end
+
+  private_class_method :new
+
+private
+
+  attr_reader :subscriber, :subscriptions
+
+  def body
+    <<~BODY
+      From now on, we'll group all the emails you get together into one digest when there's a change to content covered by one of these subscriptions:
+
+      #{subscriptions.map { |subscription| "- #{subscription.subscriber_list.title}\n" }}
+
+      If you need to keep receiving immediate emails you can change this by [managing your email subscriptions](#{manage_subscriptions_link}).
+    BODY
+  end
+
+  def manage_subscriptions_link
+    utm_source = "switch-to-daily-digest"
+    utm_medium = "email"
+    utm_campaign = "govuk-notifications"
+    utm_content = "experiment-001"
```

Comment:
Can we name the experiment at all? this feels like a rather leading number that implies a future whereas this even might be rather isolated.

Reply:
Sure, it was arbitrary: I've gone with "initial-experiment".